arbnode/espresso_force_inclusion_checker.go

```diff
@@ -123,25 +128,21 @@ func (f *ForceInclusionChecker) Start(ctx context.Context) error {
 	f.StopWaiter.Start(ctx, f)
 	var firstErrFound time.Time
 
-	// Do the check first before caff node starts
-	err := f.checkIfMessageCanBeForceIncluded(ctx)
-	if err != nil {
-		return fmt.Errorf("failed to check force inclusion: %w", err)
-	}
-
```

Comment:
These lines make sure the caff node won't be able to start if force inclusion is there. Thus, users won't use results from the caff node even for seconds or minutes. Do you think this check is unnecessary or these lines are redundant?

Reply:
I dont think we need to optimize so much, its fine. Also this causes issues in testnode because if there is any error - caff node doesnt even retry.